.github/workflows/rancher-integration.yaml

```diff
@@ -0,0 +1,194 @@
+name: Rancher integration
+
+on:
+  workflow_dispatch:
+    inputs:
+      rancherVersion:
+        description: 'Rancher Version'
+        required: true
+      rke2Version:
+        description: 'RKE2 Version'
+        required: true
+
+env:
+  LIBVIRT_DEFAULT_URI: "qemu:///system"
+
+jobs:
+  main:
+    name: Build and deploy
+    runs-on:
+      - self-hosted
+      - Linux
+      - kvm
+      - vagrant
+      - equinix
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Patch RKE2 Version
+        run: |
+          sed -i 's/^RKE2_VERSION=".*"/RKE2_VERSION="${{ github.event.inputs.rke2Version }}"/' ./scripts/version-rke2
+
+      - name: Patch Rancher Version
+        run: |
+          sed -i 's/^RANCHER_VERSION=".*"/RANCHER_VERSION="${{ github.event.inputs.rke2Version }}"/' ./scripts/version-rancher
+          sed -i 's/^  rancherImageTag: .*/  rancherImageTag: ${{ github.event.inputs.rancherVersion }}/' ./package/harvester-os/files/usr/share/rancher/rancherd/config.yaml.d/50-defaults.yaml
+          sed -i 's|^rancherInstallerImage: .*|rancherInstallerImage: rancher/system-agent-installer-rancher:${{ github.event.inputs.rancherVersion }}|' ./package/harvester-os/files/usr/share/rancher/rancherd/config.yaml.d/50-defaults.yaml
+
+      - name: Make a commit
```

Comment:
Do we need this step? Or if not commit the changed file, the following make will encounter issues? for example, getting version

Reply:
I would like to make sure there is no "-dirty" in the version name.

https://github.com/harvester/harvester-installer/blob/6973fcdfca330bb1920710f2620b6e71a8496e76/scripts/version#L3-L14

Reply:
This is what I assumed due to version.